src/algorithms/primeFinder.js

```diff
@@ -0,0 +1,19 @@
+export default function isPrime(number) {
+    var start = Math.ceil(number / 2);
```

Comment:
Use floor instead? If num/2 = 3.5 we know that 4 isn't a divisor, right?
```suggestion
    var start = Math.floor(number / 2);
```